src/utils/SolutionsUtils.js

```diff
@@ -55,9 +56,26 @@ const patchSolutionIfLocalConfigExists = async (originalSolution) => {
   ConfigUtils.patchSolution(originalSolution, overridingSolution);
 };
 
+const checkParametersValidationConstraintsInSolution = (data) => {
+  const parametersWithConstraint = data?.parameters?.filter((parameter) => parameter.options?.validation);
+  parametersWithConstraint?.forEach((parameter) => {
+    const constraint = ParameterConstraintsUtils.getParameterValidationConstraint(
+      parameter.options.validation,
+      parameter.varType,
+      data.parameters
+    );
+    if (constraint === null)
```

Comment:
When the constraint configuration is incorrect, maybe we should also delete the constraint from the config and "patch" the solution that will be used in the webapp? This could prevent some bugs with RHF trying to read invalid constraints